#tr1:
  joint_limits:

    JointRightShoulderPan:
      has_position_limits: true
      min_position: -1.0
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 0.2932
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointRightShoulderTilt:
      has_position_limits: true
      min_position: -1.4
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 0.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointRightUpperArmRoll:
      has_position_limits: true
      min_position: -3.0000
      max_position: 3.0000
      has_velocity_limits: true
      max_velocity: 0.25
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointRightElbowFlex:
      has_position_limits: true
      min_position: -1.5708
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 0.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointRightForearmRoll:
      has_position_limits: true
      min_position: -3.0000
      max_position: 3.0000
      has_velocity_limits: true
      max_velocity: 1.4661
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointRightWristFlex:
      has_position_limits: true
      min_position: -1.75
      max_position: 1.75
      has_velocity_limits: true
      max_velocity: 0.7330
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointRightWristRoll:
      has_position_limits: true
      min_position: -2
      max_position: 2
      has_velocity_limits: true
      max_velocity: 7.5398
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.5708
      min_effort: -1.5708

    JointRightGripper:
      has_position_limits: true
      min_position: 0
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftShoulderPan:
      has_position_limits: true
      min_position: -1.0
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftShoulderTilt:
      has_position_limits: true
      min_position: -1.5708
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftUpperArmRoll:
      has_position_limits: true
      min_position: -3.0000
      max_position: 3.0000
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftElbowFlex:
      has_position_limits: true
      min_position: -1.75
      max_position: 1.75
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftForearmRoll:
      has_position_limits: true
      min_position: -3.0000
      max_position: 3.0000
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftWristFlex:
      has_position_limits: true
      min_position: -1.75
      max_position: 1.75
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftWristRoll:
      has_position_limits: true
      min_position: -2
      max_position: 2
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointLeftGripper:
      has_position_limits: true
      min_position: 0
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 2.0
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0
